Packs/Securonix/Classifiers/classifier-Securonix_Incident_-_Incoming_Mapper.json

```diff
@@ -118,12 +118,32 @@
                 },
                 "Securonix Policies": {
                     "complex": {
-                        "accessor": "Policies",
                         "filters": [],
-                        "root": "reason",
+                        "root": "policy_list",
```

Comment:
Is the mapped data different than what it was? If so do we expect this change to affect the existing Securonix playbooks?

Reply:
Hi @idovandijk, there is a change in the structure of "Policies" from "reason" super key.

On previous, the "Policies" field was giving the list of policies associated with the incident.

Now, it gives the stage wise policies which is a JSON structure. To handle this scenario, we have made changes in code to directly provide the list of policies in "policy_list" field.